packages/block-library/src/table-of-contents/index.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Adding aria-label to the `core/table-of-contents` block content.
```

Comment:
```suggestion
 * Server-side rendering of the `core/table-of-contents` block.
```

Nit: To maintain consistency with other blocks